packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -5,13 +5,27 @@
 import {HttpServer} from '../../';
 import {supertest, expect} from '@loopback/testlab';
 import * as makeRequest from 'request-promise-native';
-import {ServerRequest, ServerResponse} from 'http';
+import {ServerRequest, ServerResponse, get} from 'http';
 
 describe('HttpServer (integration)', () => {
   let server: HttpServer | undefined;
 
   afterEach(stopServer);
 
+  it('formats IPv6 url correctly', async () => {
+    server = new HttpServer(dummyRequestHandler);
+    await server.start();
+    if (server.address.family === 'IPv6') {
+      const url = server.url;
+      return new Promise((resolve, reject) => {
+        get(url, res => {
+          if (res.statusCode === 200) resolve();
+          else reject();
+        });
+      });
```

Comment:
Please add `on('error')` handler, see https://nodejs.org/api/http.html#http_http_get_options_callback

```ts
get(...)
  .on('error', reject);
```

When your test fails, it does not provide any explanation. Please rework it to say what went wrong. For example:

```ts
// in our test
const response = await getAsync(server.url);
expect(response.statusCode).to.equal(200);

// implement "getAsync" helper outside the tests
function getAsync(url) {
  return new Promise((resolve, reject) => {
    get(url, resolve).on('error', reject);
  });
}
```